### PR TITLE
Enabling resnet101, vgg16, vgg19 INT8v2 model tests

### PR DESCRIPTION
**Pull request by lidanqing-intel**
1. resnet101, vgg16, vgg19, works well on current Paddle develop branch
2. googlenet will work after merging [PR17292](https://github.com/PaddlePaddle/Paddle/pull/17292)
3. mobilenetv2 will work after merging [PR17130](https://github.com/PaddlePaddle/Paddle/pull/17130)

I have tested these five models locally, all achieved good accuracy and performance. Shufflenetv2 is WIP. Please do not merge this PR before merging above-mentioned PRs.

### CODE REVIEW

**Comment by luotao1** [comment]
> resnet101, vgg16, vgg19, works well on current Paddle develop branch

Please only enable these 4 unit-tests in this PR to merge them first.

> googlenet will work after merging PR17292

Please add googlenet UT in PR17292 to merge it independently.

> mobilenetv2 will work after merging PR17130

Please add mobilenetv2 UT in PR17130

> all achieved good accuracy and performance.

Please give the result of accuracy and performance in related PR's description.

**Comment by lidanqing-vv** [comment]
> > resnet101, vgg16, vgg19, works well on current Paddle develop branch
> 
> Please only enable these 4 unit-tests in this PR to merge them first.
> 
> > googlenet will work after merging PR17292
> 
> Please add googlenet UT in PR17292 to merge it independently.
> 
> > mobilenetv2 will work after merging PR17130
> 
> Please add mobilenetv2 UT in PR17130
> 
> > all achieved good accuracy and performance.
> 
> Please give the result of accuracy and performance in related PR's description.

**Comment by lidanqing-vv** [comment]
> Please give the result of accuracy and performance in related PR's description.

Hi, @luotao1 Should I send email to you or put accuracy and performance results here directly?

**Comment by luotao1** [comment]
I get your email, thanks very much!

**Comment by lidanqing-vv** [comment]
> I get your email, thanks very much!

Could you please change one of the model name on the server under int8 folder, change `Res101_int8_model.tar.gz` to `resnet101_int8_model.tar.gz`
Really sorry, it was my typo. Thank you very much !

**Comment by luotao1** [comment]
Why do you change the name?
I see https://github.com/PaddlePaddle/Paddle/pull/17468/commits/b7de8577c1f7bfd356e9032b2580ac0adec1045b pass all the CI. You can use `Res101_int8_model.tar.gz` as well.

File: paddle/fluid/inference/tests/api/CMakeLists.txt [hunk 1]
```diff
@@ -173,6 +173,28 @@ if(WITH_MKLDNN)
     inference_download_and_uncompress(${INT8_MOBILENET_MODEL_DIR} "${INFERENCE_URL}/int8" "mobilenetv1_int8_model.tar.gz" )
   endif()
   inference_analysis_api_int8_test(test_analyzer_int8_mobilenet ${INT8_MOBILENET_MODEL_DIR} ${INT8_DATA_DIR} analyzer_int8_image_classification_tester.cc SERIAL)
+ 
+  #resnet101 int8
+  set(INT8_RESNET101_MODEL_DIR "${INT8_DATA_DIR}/resnet101")
+  if (NOT EXISTS ${INT8_RESNET101_MODEL_DIR})
+    inference_download_and_uncompress(${INT8_RESNET101_MODEL_DIR} "${INFERENCE_URL}/int8" "resnet101_int8_model.tar.gz" )
+  endif()
+  inference_analysis_api_int8_test(test_analyzer_int8_resnet101 ${INT8_RESNET101_MODEL_DIR} ${INT8_DATA_DIR} analyzer_int8_image_classification_tester.cc SERIAL)
```
Comment (luotao1) [inline paddle/fluid/inference/tests/api/CMakeLists.txt]: Please remove `SERIAL` here, see #17500

Comment (lidanqing-vv) [reply paddle/fluid/inference/tests/api/CMakeLists.txt]: > Please remove `SERIAL` here, see #17500
Ok

**Comment by lidanqing-vv** [comment]
> Why do you change the name?
> I see [b7de857](https://github.com/PaddlePaddle/Paddle/commit/b7de8577c1f7bfd356e9032b2580ac0adec1045b) pass all the CI. You can use `Res101_int8_model.tar.gz` as well.

Ok, Thanks. I will change the name back and remove SERIAL

**Comment by luotao1** [comment]
> googlenet will work after merging PR17292
mobilenetv2 will work after merging PR17130

Since #17130 is merged, could you update #17292 and add googlenet and mobilenetv2 ut at first?

**Pull request merged**